testdata/p4_16_errors_outputs/issue2544_recursive.p4-stderr

```diff
@@ -0,0 +1,6 @@
+issue2544_recursive.p4(2): [--Werror=invalid] error: Recursive declaration of 'S' in 'struct S'
```

Comment:
This message isn't appropriate here either. This is not a declaration of S, but of x, so it is not really a recursive declaration.